### DISCUSSION TITLE
Considerations before deploying T-POT in a Cloud & Upgrade from 22.04 to 24.04

### DISCUSSION THREAD

**Discussion by IKMAR00** [telekom-security/tpotce#1498] [Q&A]
Hello,

I am new to honeypots, and I want to deploy T-POT in a cloud environment.

I would like to know more about what considerations should I address before deploying T-POT in a cloud environment, because, as far as I know, certain cloud providers could tag you as a malicious user if you deploy a honeypot without a prior notification and could result in account suspension.

Another topic I want to address is that T-POT 24.04 is going to be released soon, and I am wondering what this means for a previous 22.04 installation, other than the end of support.

For example, if a user has a server with an Intel machine that supports the minimum installation requirements for version 22.04 (8GB RAM, 128 GB SSD) and upgrades to 24.04, will the previously collected data and settings be present after the update?

Thank you and I will be awaiting your response.

**Comment by t3chn0m4g3** [comment]
#1487

**Reply by IKMAR00** [reply]
Okay.
Reading the announcement again makes it clear that there is no possibility of upgrading a previous installation, and you must reinstall the new version of T-POT.

As for cloud deployment, I am not asking about what settings a user needs to make to be able to run T-POT in that environment.

What I intended to ask is if there is a specific process that a user must perform before being allowed to run T-POT in the cloud and not be labeled as a malicious user.

Thanks for your time!
